sale_exception/tests/test_sale_exception.py

```diff
@@ -204,3 +204,18 @@ def test_exception_no_free(self):
         so_except_confirm.action_confirm()
         self.assertFalse(sale_order.ignore_exception)
         self.assertTrue(sale_order.state == "draft")
+
+    def test_exception_no_stock_warning(self):
```

Comment:
```suggestion
    def test_exception_no_validation_for_unlock(self):
```
or whatever that describes the test scenario better.

Reply:
Yes! This name makes sense for this situation